src/pages/projects.js

```diff
@@ -508,6 +508,33 @@ const ProjectDescriptions = () => (
         </div>
       </div>
     </div>
+
+    <div className="project-box">
+      <a name="tracee" />
+      <div className="project-major-title">Tracee</div>
+      <div className="project-minor-title">Dynamic Security Event Tracing</div>
+      <div className="project-body">
+        <a
+          className="project-logo"
+          target="_blank"
+          href="https://github.com/aquasecurity/tracee"
+        >
+          <img src="https://raw.githubusercontent.com/aquasecurity/tracee/master/images/tracee.png" />
+        </a>
+        <div className="project-description">
+          <p>
+            <a href="https://github.com/aquasecurity/tracee">
+              <b>GitHub</b>
+            </a>{" "}
+          </p>
+          <p>
+            Tracee is a lightweight, efficient and easy-to-use tool for dynamically detecting security events from systems, containers, and pods. 
```

Comment:
Maybe best to stick to concrete and specific adjectives here to describe Tracee? All of the above projects could probably be described as lightweight or efficient in some sense, and hopefully none would qualify themselves as hard to use :slightly_smiling_face: 

One way to address this would be to replace the adjectives with Tracee's feature or property that motivated you to use that adjective. For example, `lightweight and easy-to-use` could be `Tracee is packaged as a single Docker image.`

Reply:
Nit: Also, serial comma rule isn't consistent here. It should be `lightweight, efficient, and easy-to-use tool`.

Reply:
I see you're a fan of the Oxford comma! Point taken on the phrasing, we will come back with a better description.

Reply:
I've updated the description

Reply:
> I see you're a fan of the Oxford comma!

No no, no specific preference here other than consistency. Same as for spaces vs. tabs I guess :smile: